website/content/api-docs/secret/pki.mdx

```diff
@@ -16,1178 +16,1121 @@ update your API calls accordingly.
+These are unauthenticated endpoints.
 
-- `serial` `(string: <required>)` – Specifies the serial of the key to read.
-  This is part of the request URL. Valid values for `serial` are:
+| Method | Path                           | Issuer    | Format |
+| :----- | :----------------------------- | :-------- | :----- |
+| `GET`  | `/pki/cert/ca`                 | `default` | JSON   |
+| `GET`  | `/pki/ca`                      | `default` | DER    |
+| `GET`  | `/pki/ca/pem`                  | `default` | PEM    |
+| `GET`  | `/pki/issuer/:issuer_ref/json` | Selected  | JSON   |
+| `GET`  | `/pki/issuer/:issuer_ref/der`  | Selected  | DER    |
+| `GET`  | `/pki/issuer/:issuer_ref/pem`  | Selected  | PEM    |
 
-  - `<serial>` for the certificate with the given serial number
-  - `ca` for the CA certificate
-  - `crl` for the current CRL
-  - `ca_chain` for the CA trust chain; intermediate certificates in the `certificate`
-               field, full chain in the `ca_chain` field.
+#### Parameters
 
-### Sample Request
+- `issuer_ref` `(string: <required>)` - Reference to an existing issuer,
+  either by Vault-generated identifier, the literal string `default` to
+  refer to the currently configured default issuer, or the name assigned
+  to an issuer. This parameter is part of the request URL.
+
+~> Note: This parameter is not present on the `/pki/cert/ca` and
+   `/pki/ca(/pem)?` paths and takes the implicit value `default`.
+
+#### Sample Request
 
 ```shell-session
 $ curl \
-    http://127.0.0.1:8200/v1/pki/cert/crl
+    http://127.0.0.1:8200/v1/pki/issuer/root-x1/json
 ```
 
-### Sample Response
+#### Sample Response
 
-```json
+```text
 {
   "data": {
-    "certificate": "-----BEGIN CERTIFICATE-----\nMIIGmDCCBYCgAwIBAgIHBzEB3fTzhTANBgkqhkiG9w0BAQsFADCBjDELMAkGA1UE\n..."
+    "ca_chain": [
+      "-----BEGIN CERTIFICATE-----\nMIIDFDCCAfygAwIBAgIUXgxy54mKooz5soqQoRINazH/3pQwDQYJKoZIhvcNAQEL\n...",
+      "-----BEGIN CERTIFICATE-----\nMIIDFTCCAf2gAwIBAgIUUo/qwLm5AyqUWqFHw1MlgwUtS/kwDQYJKoZIhvcNAQEL\n..."
+    ],
+    "certificate": "-----BEGIN CERTIFICATE-----\nnMIIDFDCCAfygAwIBAgIUXgxy54mKooz5soqQoRINazH/3pQwDQYJKoZIhvcNAQEL...",
+    "revocation_time": 0
   }
 }
 ```
 
-## Read Raw Certificate
+<a name="read-ca-certificate-chain"></a>
 
-This endpoint retrieves one of a selection of certificates _in raw DER-encoded
-form_. This is a bare endpoint that does not return a standard Vault data
-structure and cannot be read by the Vault CLI; use `/pki/cert/:serial` for that. If
-`/pem` is added to the endpoint, the selected certificate is returned in PEM format.
+### Read Default Issuer Certificate Chain
 
-This is an unauthenticated endpoint.
+This endpoint retrieves the default issuer's CA certificate chain, including
+the default issuer. When the format of the endpoint is notated below as JSON,
+this is a standard Vault data structure that can be read by the Vault CLI.
+Otherwise, the PEM format cannot be read via the `vault read` CLI; another
```

Comment:
I wonder instead of repeating the same blurb about “When the format is notated below as JSON, …” across the various api endpoints, if we have some sort of [1] link next to the DER/PEM format table entries that brings people to the message?